exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -218,11 +219,12 @@ def __init__(
 
         parsed_url = urlparse(endpoint)
 
-        if insecure is None:
-            if parsed_url.scheme == "https":
-                insecure = False
-            else:
-                insecure = True
+        insecure = insecure or environ.get(OTEL_EXPORTER_OTLP_INSECURE, False)
+
+        if insecure and parsed_url.scheme != "https":
```

Comment:
What is insecure is not set and scheme is `http`? This is one of the reason tests are failing

Reply:
Thanks for the continued help.

I have reverted the default to `insecure=None`, and have the scheme decide `insecure` in these cases, defaulting to secure.

Now if `insecure=None` and scheme is `http`, we get insecure.